v1/test/env.spec.ts

```diff
@@ -36,12 +37,12 @@ describe("applyEnvVariables", () => {
       Environment: {
         Variables: {
           [DD_HANDLER_ENV_VAR]: "hello.handler",
-          [FLUSH_METRICS_TO_LOGS_ENV_VAR]: transportDefaults.flushMetricsToLogs.toString(),
```

Comment:
the vast majority of unit tests in this file use "true" rather than getting the boolean string from the DefaultDatadogProps. Changed this over for the sake of consistency over this unit test file